packages/core/content-manager/server/src/history/controllers/history-version.ts

```diff
@@ -4,6 +4,29 @@ import { getService as getContentManagerService } from '../../utils';
 import { getService } from '../utils';
 import { HistoryVersions } from '../../../../shared/contracts';
 
+const validatePagination = ({ page, pageSize }: { page: any; pageSize: any }) => {
+  let pageNumber = 1;
+  let pageSizeNumber = 10;
+
+  if (page) {
+    pageNumber = parseInt(page, 10);
+
+    if (Number.isNaN(pageNumber) || pageNumber < 1) {
+      throw new errors.PaginationError('invalid pageNumber param');
+    }
+  }
+
+  if (pageSize) {
+    pageSizeNumber = parseInt(pageSize, 10);
+
+    if (Number.isNaN(pageSizeNumber) || pageSizeNumber < 1 || pageSizeNumber > 100) {
+      throw new errors.PaginationError('invalid pageSize param');
+    }
+  }
```

Comment:
I'm not sure I understand why we need to throw an error for invalid pagination in a controller. If something is not what we expect couldn't we just fallback to a default?